scripts/RT/wepp_worker.py

```
@@ -41,7 +42,7 @@ def run_wepp(payload: WeppJobPayload):
     # We run timeout to keep things from hanging indefinitely, we tried 60
     # seconds but it was too short as sometimes latency happens.
     with subprocess.Popen(
-        ["timeout", "-s", "9", "600", payload.weppexe],
+        ["timeout", "-s", "9", "600", sanitize_exe(payload.weppexe)],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)